runtime/autoload/man.vim

```diff
@@ -388,4 +387,14 @@ function! man#init_pager() abort
   execute 'silent file man://'.fnameescape(ref)
 endfunction
 
+function! man#highlight_formatted_text() abort
+  let l:modifiable = &modifiable
+  set modifiable
```

Comment:
This could be done in the Lua code via `vim.api.nvim_command()` or whatever. Then we can eliminate this "public" VimL function.